app/controllers/comments_controller.rb

```diff
@@ -33,6 +34,15 @@ def destroy
 
   private
 
+  def create_notification(post, comment)
+    return if post.user.id == current_user.id
+    Notification.create(user_id: post.user.id,
+                        notified_by_id: current_user.id,
+                        post_id: post.id,
+                        identifier: comment.id,
+                        notice_type: "comment")
```

Comment:
You should build params and call here

Reply:
Sorry I am a bit confusing...
Should i use JSON params or params permit?

Reply:
Ok no problem, you can use current code